tests/loaders/test_overlay.py

```diff
@@ -0,0 +1,22 @@
+from dissect.target import Target
+from dissect.target.filesystem import VirtualFilesystem
+from dissect.target.loaders.overlay import Overlay2Loader
+from tests.plugins.apps.container.test_docker import (  # noqa: F401
+    fs_docker,
+    target_linux_docker,
+)
+
+
+def test_overlay_loader_docker_container(
+    target_linux_docker: Target, fs_docker: VirtualFilesystem  # noqa: F811
```

Comment:
I took a closer look, you're using pytest fixtures not quite how they're supposed to be used. You don't have to manually import them, pytest will insert the correct fixture by name of the argument. You also don't directly use the `fs_docker` fixture in your test, so there's no need to reference it, it's already implicitly included from `target_linux_docker`. Only if you need to interact with the fixture in your own test should you include it.

Reply:
Ah I missed that, apologies. I didn't look close enough ;) Let me come back to you on this.

Reply:
In the previous situation, the fixture was only used in that test file. In that case, it's indeed recommended to only place it in that file. However, since you're now using the fixture in different test files, it's better to move it to the global `conftest.py`, rather than doing an importing dance. `conftest.py` is also directory scoped, but unfortunately that doesn't help us in this case.